openmdao/docs/usr-guide/tutorials/external-code-tutorial.rst

```diff
@@ -43,13 +43,16 @@ and writes the output, `f_xy`, to an output file.
 
         input_filename = sys.argv[1]
         output_filename = sys.argv[2]
-        
+
         paraboloid(input_filename, output_filename)
 
 
-Next we need to write the code that makes use of this external code.
 
-::
+Next we need to build the OpenMDAO component that makes use of this external code.
+
+
+.. code-block :: python
```

Comment:
So, not using doctests? I'd recommend at least instantiating the component you build so that it flags any syntax errors.